Runtime/AvatarCreator/WebRequests/PartnerAssetsRequests.cs

```diff
@@ -69,6 +69,48 @@ public async Task<PartnerAsset[]> Get(BodyType bodyType, OutfitGender gender, Ca
             return assets.ToArray();
         }
 
+        public async Task<PartnerAsset[]> GetCategory(Category category, BodyType bodyType, OutfitGender gender, CancellationToken ctx = new CancellationToken())
+        {
+            var assets = new HashSet<PartnerAsset>();
+            AssetLibrary assetLibrary;
+
+            try
+            {
+                assetLibrary = await GetRequest(LIMIT, 1, category, gender, bodyType, ctx: ctx);
```

Comment:
I know it's not in your PR, but can we rename 'GetRequest' to something more explicit, as it currently means nothing 😅 - maybe 'ListAssetsRequest'?